api/provider/aws/test_aws/environment_read_test.go

```diff
@@ -93,9 +95,11 @@ func TestEnvironmentRead(t *testing.T) {
 		EnvironmentID:   "env_id",
 		EnvironmentName: "env_name",
 		OperatingSystem: "linux",
-		InstanceSize:    "m3.small",
+		InstanceType:    "m3.small",
```

Comment:
I don't believe that m3.small is a valid instance type anymore.

https://aws.amazon.com/ec2/instance-types/